docs/web3.geth.rst

```diff
@@ -264,9 +264,10 @@ The following methods are available on the ``web3.geth.personal`` namespace.
 
     * Delegates to ``personal_unlockAccount`` RPC Method
 
-    Unlocks the given ``account`` for ``duration`` seconds.  If ``duration`` is
-    ``None`` then the account will remain unlocked indefinitely.  Returns
-    boolean as to whether the account was successfully unlocked.
+    Unlocks the given ``account`` for ``duration`` seconds. If ``duration`` is 
+	``None`` then the account will remain unlocked for 300 seconds (which is current default by Geth), 
```

Comment:
Can you add the version number of geth to this.

Reply:
you mean 
(which is current default by Geth) => (which is current default by Geth v1.9.5)
 right?